app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -44,7 +53,7 @@ class PatientRepository @Inject constructor(
     }
 
     return database.patientSearchDao()
-        .search(query!!)
+        .search(strip(query!!))
```

Comment:
We should also lowercase the strings here.
It's an optimization, but we can avoid computing that during search.

Reply:
The SQLite query we use is already case-insensitive (`LIKE`). This is why I didn't do it, since it seemed unnecessary.

Reply:
Resolved on a call